contracts/src/v0.8/dev/KeeperRegistrarDev.sol

```diff
@@ -302,6 +303,7 @@ contract KeeperRegistrarDev is TypeAndVersionInterface, ConfirmedOwner {
     uint256 amount,
     bytes calldata data
   ) external onlyLINK permittedFunctionsForLINK(data) isActualAmount(amount, data) isActualSender(sender, data) {
+    if (data.length < 292) revert InvalidDataLength();
```

Comment:
Can we be more strict and use `!=` ?

Reply:
nope, data contains string name & bytes encryptedEmail which can be variable length

Reply:
ah, okay :+1: